src/main/java/org/apache/sling/models/impl/ModelAdapterFactory.java

```diff
@@ -1046,11 +1075,22 @@ protected ThreadInvocationCounter initialValue() {
 
         this.configPrinterRegistration = bundleContext.registerService(Object.class.getName(),
                 new ModelConfigurationPrinter(this, bundleContext, adapterImplementations), printerProps);
+
+        Hashtable<Object, Object> listenerProps = new Hashtable<>();
+        listenerProps.put("osgi.http.whiteboard.context.select", "(osgi.http.whiteboard.context.name=*)");
```

Comment:
Could you use the constants from http://javadox.com/org.osgi/osgi.cmpn/6.0.0/org/osgi/service/http/whiteboard/HttpWhiteboardConstants.html.

Reply:
Not without bumping the dependency. But maybe that should be done anyway since (as far as I know), this functionality will effectively depend upon a 6.0 HttpService implementation.

Reply:
Yes, bumping the dependency is reasonable as HTTP whiteboard is only available since OSGi Comp R6.